### PR TITLE
Model additions to represent education and awards and risk additions

### PR DESCRIPTION
**Pull request by invisig0th**

### CODE REVIEW

File: synapse/models/person.py [hunk 1]
```diff
@@ -31,6 +43,63 @@ def getModelDefs(self):
...
+                    ('classes', ('array', {'type': 'edu:class'}), {
+                        'doc': 'The classes attended by the student',
+                    }),
+                    ('achievement', ('ps:achievement', {}), {
+                        'doc': 'The achievement awarded to the individual.',
+                    }),
+                )),
+                ('ps:achievement', {}, (
+                    ('awardee', ('ps:contact', {}), {
+                        'doc': 'The recipeient of the award.',
```
Comment (whippit) [inline synapse/models/person.py:116]: spelling

File: synapse/models/orgs.py [hunk 2]
```diff
@@ -108,8 +108,24 @@ def getModelDefs(self):
...
+                    ('name', ('str', {'lower': True, 'strip': True, 'onespace': True}), {
+                        'doc': 'The name of the award.',
+                        'ex': 'Bachelors of Science',
+                    }),
+                    ('type', ('str', {'lower': True, 'strip': True, 'onespace': True}), {
+                        'doc': 'The type of award.',
+                        'ex': 'certification',
+                    }),
+                    ('org', ('ou:org', {}), {
+                        'doc': 'The orgnazation which issues the award.',
```
Comment (therealsilence) [inline synapse/models/orgs.py:126]: "orgnazation" -> organization

File: synapse/models/person.py [hunk 3]
```diff
@@ -31,6 +43,91 @@ def getModelDefs(self):
...
+                        'doc': 'The primary instructor for the class.',
+                    }),
+                    ('assistants', ('array', {'type': 'ps:contact'}), {
+                        'doc': 'An array of assistant/co-instructor contacts.',
+                    }),
+                    ('date:first', ('time', {}), {
+                        'doc': 'The date of the first day of class.'
+                    }),
+                    ('date:last', ('time', {}), {
+                        'doc': 'The date of the first day of class.'
```
Comment (therealsilence) [inline synapse/models/person.py:79]: copypasta error - s/b "last day of class"

**Review by therealsilence — COMMENTED** [review]
Minor typos/spelling

**Review by Cisphyx — APPROVED** [review]

**Pull request merged**